cogs/birthday.py

```diff
@@ -0,0 +1,97 @@
+import datetime
+import re
+from typing import Pattern
+
+from discord.ext import commands
+
+
+class Birthdays(commands.Cog):
+    DATEPATTERN: Pattern[str] = re.compile(r'(((0[1-9])|(1[0-9])|(2[0-9])|(3[0-1]))\.'  # 01.-31.
+                                           r"((01)|(03)|(05)|(07)|(08)|(10)|(12))\.)"  # all months with 31 days
+                                           r"|"
+                                           r"(((0[1-9])|(1[0-9])|(2[0-9])|(30))\."  # 01.-30.
+                                           r"((04)|(06)|(09)|(11))\.)"  # all months with 30 days
+                                           r"|"
+                                           r"(((0[1-9])|(1[0-9])|(2[0-9]))\."  # 01.-29.
+                                           r"02\.)")  # february
+    BIRTHDAY_CHANNEL_ID = 666653781366145028
+
+    def __init__(self, bot):
+        self.bot = bot
+        self.bot.register_job(60 * 60 * 24, self.congratulate)
+
+    @commands.group(aliases=['birth', 'birthday', 'birthdate', 'geburtstag'])
+    async def birthdays(self, ctx):
```

Comment:
The `invoked_subcommand` thing isn't needed, there is a better solution: https://github.com/timschumi/TUMbot/blob/master/cogs/reactionroles.py#L12 (note the `invoke_without_command=True`)